var/spack/repos/builtin/packages/mxml/package.py

```diff
@@ -31,8 +31,7 @@ class Mxml(AutotoolsPackage):
     non-standard libraries.
     """
 
-    homepage = "http://www.msweet.org"
-    url = "http://www.msweet.org/files/project3/mxml-2.9.tar.gz"
```

Comment:
Yeah, things are pretty broken at the moment. I fixed everything in #2972. I should probably clean that up and get it merged so you won't have to add a `url_for_version` at all.